### PR TITLE
Identity admin

### PR DESCRIPTION
**Pull request by michaeldfallen**
This pull request implements pivotal story [Front End/DB/Fake identity data](https://www.pivotaltracker.com/story/show/93529766).

The changes in this pull request add a Postgres DB and a new table to store Identities, which match the basic assertions offered by GOV.UK Verify. It also provides an interface on `http://verifudge.lrdigitalmortgage.com/identity` to list and create new identities.

The plan is for a future story to use these identities to send match requests to our matching service.

### CODE REVIEW

File: app/config.py [hunk 1]
```diff
@@ -1 +1,7 @@
+import os
+
+
 DEBUG = True
+
+SQLALCHEMY_DATABASE_URI = os.getenv('DEED_DATABASE_URI',
```
Comment (dawidskainos) [inline app/config.py]: I guess this should be VERIFUDGE_DATABASE_URI (not DEED). It should be set in puppet/templates/service.systemd.erb

File: app/db/migrations/versions/1787d27cfda_.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# flake8: noqa
```
Comment (dawidskainos) [inline app/db/migrations/versions/1787d27cfda_.py:1]: Wouldn't it be better to create only one migration file? Or is there an idea behind three separate files?

File: app/config.py (hunk 1 above)
Comment (michaeldfallen) [inline app/config.py]: Nice catch. I also hadn't ran the database upgrade or passed this variable to the app. By-product of adding a DB to a frontend app.

File: app/db/migrations/versions/1787d27cfda_.py (hunk 2 above)
Comment (michaeldfallen) [inline app/db/migrations/versions/1787d27cfda_.py:1]: I didn't do this intentionally, it was a byproduct of the way I wrote the work. First I created the basic blueprint layout, then I added the DB, then I created the Identity class. Later on I added the different pieces of the Identity when I needed them. So the multiple migrations are a product of working to a goal (first "Store Identities in a DB and list them", second "Store the info needed by Verify in Identities").

In terms of whats better, meh. Shouldn't really matter either way. If we are worried about migrations not applying then we'd have bigger problems.

**Pull request merged**